### PR TITLE
Suggested Users

### PR DESCRIPTION
**Pull request by drunkleen**
### Description

This pull request adds  `get_suggested_users` to get the list of suggested users to follow.

### Changes Made

- Added the `get_suggested_users` method.
- `README.md` updated.

### Checklist
- [x] Tested the new method.
- [x] Updated the documentation to include the new method.

### CODE REVIEW

**Comment by Mineru98** [comment]
Could you write it with the test code?

**Review by junhoyeo — APPROVED** [review]
Code itself looks clean for me!

File: threadspy/threads_api.py [hunk 1]
```diff
@@ -109,8 +109,14 @@ def __download(self, url: str) -> bytes:
             return None
 
     def __get_app_headers(self) -> dict:
+        """
+        Generates Header App.
```
Comment (junhoyeo) [inline threadspy/threads_api.py]:
```suggestion
        Generates App Headers.
```

File: README.md [hunk 2]
```diff
@@ -165,7 +165,7 @@ user_id = api.get_user_id_from_username('junhoyeo')
 api.friendship_status(user_id)
 ```
 
-#### 🧑‍🦳 Get User Followings and Followers
+#### 👉🏻👈🏻 Get User Followings and Followers
```
Comment (junhoyeo) [inline README.md:168]: 😆 Cute 👍👍

**Comment by Mineru98** [comment]
I validated this code with test code.

**Pull request merged**